[2024-11-03T23:42:57.162+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_feature_select_and_data_augmentation.feature_selection_task manual__2024-11-03T23:42:54.803351+00:00 [queued]>
[2024-11-03T23:42:57.166+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_feature_select_and_data_augmentation.feature_selection_task manual__2024-11-03T23:42:54.803351+00:00 [queued]>
[2024-11-03T23:42:57.166+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-11-03T23:42:57.166+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-11-03T23:42:57.166+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-11-03T23:42:57.172+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): feature_selection_task> on 2024-11-03 23:42:54.803351+00:00
[2024-11-03T23:42:57.174+0000] {standard_task_runner.py:55} INFO - Started process 169 to run task
[2024-11-03T23:42:57.176+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DAG_feature_select_and_data_augmentation', 'feature_selection_task', 'manual__2024-11-03T23:42:54.803351+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/feature_and_augm_dag.py', '--cfg-path', '/tmp/tmpyzfdj_r6']
[2024-11-03T23:42:57.177+0000] {standard_task_runner.py:83} INFO - Job 123: Subtask feature_selection_task
[2024-11-03T23:42:57.202+0000] {task_command.py:388} INFO - Running <TaskInstance: DAG_feature_select_and_data_augmentation.feature_selection_task manual__2024-11-03T23:42:54.803351+00:00 [running]> on host b77d411e2e76
[2024-11-03T23:42:57.229+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=House_Price_Prediction Team
AIRFLOW_CTX_DAG_ID=DAG_feature_select_and_data_augmentation
AIRFLOW_CTX_TASK_ID=feature_selection_task
AIRFLOW_CTX_EXECUTION_DATE=2024-11-03T23:42:54.803351+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-03T23:42:54.803351+00:00
[2024-11-03T23:42:57.233+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/feature_and_augm_dag.py", line 59, in feature_selection_callable
    raise ValueError("No encoded data found in XCom for 'encoded_data'")
ValueError: No encoded data found in XCom for 'encoded_data'
[2024-11-03T23:42:57.236+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=DAG_feature_select_and_data_augmentation, task_id=feature_selection_task, execution_date=20241103T234254, start_date=20241103T234257, end_date=20241103T234257
[2024-11-03T23:42:57.240+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 123 for task feature_selection_task (No encoded data found in XCom for 'encoded_data'; 169)
[2024-11-03T23:42:57.276+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-11-03T23:42:57.289+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
